hw/drivers/trng/trng_da1469x/src/trng_da1469x.c

```diff
@@ -105,3 +106,20 @@ da1469x_trng_init(struct os_dev *dev, void *arg)
 
     return 0;
 }
+
+#ifdef MBEDTLS_ENTROPY_HARDWARE_ALT
+int
+mbedtls_hardware_poll( void *data, unsigned char *output, size_t len, size_t *olen ) {
+    struct trng_dev *trng;
+
+    trng = (struct trng_dev *)os_dev_lookup("trng");
+    da1469x_trng_init((struct os_dev *)trng, NULL);
+    int ret = da1469x_trng_read(trng, output, len);
```

Comment:
in mynewt variable declaration are not placed in the middle of block

Reply:
will change it